Resources/Locale/en-US/_NF/bluespace-events/events.ftl

```diff
@@ -1,8 +1,8 @@
-station-event-bluespace-vault-start-announcement = A NanoTrasen autonomous armored vault transport has made an FTL error and will soon arrive nearby. Your sector will be rewarded for it's safe return.
-station-event-bluespace-vault-end-announcement = We have successfully retrieved the vault from your sector and have reimbursed your nearby Frontier Outpost accordingly.
+station-event-bluespace-vault-start-announcement = A NanoTrasen armored vault is inbound to the sector. The area 300m around the vault is a No Fly Zone. Station accounts will be rewarded for its safe return.
```

Comment:
```suggestion
station-event-bluespace-vault-start-announcement = A NanoTrasen armored vault is inbound to the sector. The NFSD may declare the vault a No Fly Zone. Station accounts will be rewarded for its safe return.
```

Reply:
With Dvir's change requiring that a Sheriff is present for these events to spawn, I feel like having a static number is fine. Sheriff bein on usually means a increase in NFSD

Reply:
I am still of the opinion that this should be their call - do not make a call for them, _that is their job_.

Reply:
i think I agree with whatston3 here, let the sheriff decide. If they are occupied with a pirate chase or simply dont have the numbers to respond, or any other reasons, they could choose not to guard the cache etc.